media/css/firefox/whatsnew/whatsnew-142.scss

```diff
@@ -0,0 +1,866 @@
+    box-shadow: var(--fds-shadow-lg);
+  }
+
+  &:active {
+    transform: translateY(0);
+  }
+
+  &.wnp-button-secondary {
+    background: var(--fds-color-secondary);
+
+    &:hover {
+      background: var(--fds-color-secondary-dark);
+    }
+  }
+
+  &.wnp-button-outline {
+    background: transparent;
+    color: var(--fds-color-gray-90);
+    position: relative;
+
+    &::before {
+      content: "";
+      position: absolute;
+      inset: 0;
+      border: 1px solid var(--fds-color-gray-90);
+      border-radius: inherit;
+      pointer-events: none;
+    }
+
+    &:hover::before {
+      border-color: var(--fds-color-secondary-dark);
+    }
+  }
+}
+
+.wnp-card {
+  background: var(--fds-color-ash);
+  border: 1px solid var(--fds-color-gray-30);
+  border-radius: var(--fds-border-radius-lg);
+  padding: var(--fds-spacing-xl);
+  transition: all var(--fds-transition-base);
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  gap: var(--fds-spacing-md);
+
+  &:hover {
+    transform: translateY(-2px);
+    box-shadow: var(--fds-shadow-xl);
+  }
+}
+
+.wnp-footer {
+  margin-top: auto;
+  padding: var(--fds-spacing-4xl) 0;
+  background: var(--fds-color-ash);
+  border-top: 1px solid var(--fds-color-gray-30);
+}
+
+.wnp-footer-inner {
+  max-width: 1280px;
+  margin: 0 auto;
+  padding: 0 var(--fds-spacing-4xl);
+  color: var(--fds-color-gray-60);
+  font-size: var(--fds-font-size-sm);
+}
+
+.wnp-footer-top {
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+  gap: var(--fds-spacing-xl);
+}
+
+.wnp-footer-copy {
+  margin: 0;
+  color: var(--fds-color-gray-70);
+}
+
+.wnp-footer-link {
+  color: var(--fds-color-info);
+  text-decoration: underline;
+}
+
+.wnp-footer-logo {
+  width: 200px;
+  height: 66px;
+  background: var(--fds-color-ash);
+  border-radius: var(--fds-border-radius-sm);
+}
+
+.wnp-footer-divider {
+  border: none;
+  border-top: 1px solid var(--fds-color-gray-30);
+  margin: var(--fds-spacing-3xl) 0;
+}
+
+.wnp-footer-bottom {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  gap: var(--fds-spacing-xl);
+}
+
+.wnp-footer-title {
+  font-family: var(--fds-font-family-primary);
+  font-size: var(--fds-font-size-2xl);
+  color: var(--fds-color-gray-90);
+  margin: 0 0 var(--fds-spacing-xs);
+}
+
+.wnp-footer-subtitle {
+  margin: 0;
+  color: var(--fds-color-gray-70);
+}
+
+.wnp-footer-cta {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: var(--fds-spacing-sm);
+  width: 308px;
+}
+
+.wnp-footer-note {
+  margin: 0;
+  color: var(--fds-color-gray-70);
+  font-size: var(--fds-font-size-xs);
+}
+
+.wnp-eyebrow {
+  font-family: var(--fds-font-family-secondary);
+  font-size: var(--fds-font-size-sm);
+  color: var(--fds-color-gray-60);
+  margin-bottom: var(--fds-spacing-sm);
+  text-align: center;
+}
+
+.wnp-spotlight {
+  background-color: transparent;
+
+  .wnp-container {
+    position: relative;
```

Comment:
Wondering, is your IDE picking up .editorconfig? I think we specifically add double spacing for CSS/SASS.

Reply:
(Linter should not care too much, but the next person editing it might have unexpected whitespace changes saving it.)